event/events.go

```diff
@@ -0,0 +1,20 @@
+package event
+
+// NewInstanceEvent represents a 'Dimensions Extracted' kafka messagae.
+type NewInstanceEvent struct {
```

Comment:
consider removing 'event' from the end of the names here, as its part of the event package it will already have the 'context' of an event.